### DISCUSSION TITLE
The future of Lime3DS - What exactly will happen after the merger?

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [azahar-emu/azahar#532] [Q&A] [answered]
I'm hoping the merger will not backfire much like the Sprint-Nextel merger did. But what exactly will happen to Lime3DS after the fact? Will a link be added to it's successor and the repo archived? Or will the whole repo get deleted? I'm hoping it's the former that will happen.

**Comment by OpenSauce04** [comment, marked as answer]
The current plan is for the new project to occupy the current Lime3DS repository in order to maintain currently open issues and pull requests, alongside making it easier for people to discover the new project. The old Lime3DS code will be archived in the form of either a branch or a separate repo, although this has not yet been decided.
